tests/test_file.py

```diff
@@ -382,6 +382,47 @@ def _test_file_get_latest(self, replica: Replica):
                 return
         self.fail(f"Failed after {FILE_GET_RETRY_COUNT} retries.")
 
+    def test_file_get_direct(self):
+        self._test_file_get_direct(Replica.aws)
+        self._test_file_get_direct(Replica.gcp)
+
+    def _test_file_get_direct(self, replica: Replica):
+        """
+        Verify that the direct URL option works for GET/ file
+        """
+        file_uuid = "ce55fd51-7833-469b-be0b-5da88ebebfcd"
+
+        native_url = str(UrlBuilder()
+                         .set(path="/v1/files/" + file_uuid)
+                         .add_query("replica", replica.name)
+                         .add_query("directurl", "True"))
+        presigned_url = str(UrlBuilder()
+                            .set(path="/v1/files/" + file_uuid)
+                            .add_query("replica", replica.name))
+        with override_bucket_config(BucketConfig.TEST_FIXTURE):
+            native_resp_obj = self.assertGetResponse(
```

Comment:
Additional/different testing is needed. Let's talk about this.

Reply:
Added tests with blob_size from native url

Reply:
added tests for schema + checkout bucket in url